packages/maskbook/src/plugins/Polls/index.ts

```diff
@@ -0,0 +1,11 @@
+import { registerPlugin } from '@masknet/plugin-infra'
+import { base } from './base'
+
+registerPlugin({
+    ...base,
+    SNSAdaptor: {
```

Comment:
IIRC this plugin definitely requires a Worker entry to work.

Reply:
Is this "empty" worker correct? https://github.com/DimensionDev/Maskbook/pull/3500/commits/21b352cf9530132b9f63c8d48666d2cdf87cf240

Reply:
Hmm, I found this plugin is a bit complicated than others, let me handle this one.